Schweizerisches Bundesgericht.js

```diff
@@ -0,0 +1,411 @@
+				"docketNumber": "BGE 81 I 63",
+				"url": "https://www.bger.ch/ext/eurospider/live/de/php/clir/http/index.php?lang=de&type=highlight_simple_query&page=1&from_date=&to_date=&from_year=1954&to_year=2021&sort=relevance&insertion_date=&from_date_push=&top_subcollection_clir=bge&query_words=UNterhalt&part=all&de_fr=&de_it=&fr_de=&fr_it=&it_de=&it_fr=&orig=&translation=&rank=1&highlight_docid=atf%3A%2F%2F81-I-63%3Ade&number_of_ranks=714&azaclir=clir",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.bger.ch/ext/eurospider/live/de/php/aza/http/index_aza.php?date=20211116&lang=de&mode=news",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://www.bger.ch/ext/eurospider/live/de/php/aza/http/index.php?highlight_docid=aza%3A%2F%2Faza://27-10-2021-8C_329-2021&lang=de&zoom=&type=show_document",
+		"items": [
+			{
+				"itemType": "case",
+				"caseName": "BGer 8C_329/2021",
+				"creators": [],
+				"dateDecided": "2021-10-27",
+				"court": "Schweizerisches Bundesgericht",
+				"docketNumber": "8C_329/2021",
+				"url": "https://www.bger.ch/ext/eurospider/live/de/php/aza/http/index.php?highlight_docid=aza%3A%2F%2Faza://27-10-2021-8C_329-2021&lang=de&zoom=&type=show_document",
+				"attachments": [
+					{
+						"title": "Snapshot",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.bger.ch/ext/eurospider/live/de/php/aza/http/index.php?lang=de&type=simple_query&query_words=Unterhalt&lang=de&top_subcollection_aza=all&from_date=&to_date=",
+		"items": "multiple"
+	},
+	{
+		"type": "web",
+		"url": "https://www.bger.ch/ext/eurospider/live/fr/php/clir/http/index.php?lang=fr&type=highlight_simple_query&page=1&from_date=&to_date=&from_year=1954&to_year=2021&sort=relevance&insertion_date=&from_date_push=&top_subcollection_clir=bge&query_words=UNterhalt&part=all&de_fr=&de_it=&fr_de=&fr_it=&it_de=&it_fr=&orig=&translation=&rank=1&highlight_docid=atf%3A%2F%2F81-I-63%3Ade&number_of_ranks=714&azaclir=clir",
+		"items": [
+			{
+				"itemType": "case",
+				"caseName": "ATF 81 I 63",
+				"creators": [],
+				"dateDecided": 1955,
+				"abstractNote": "Militärpflichtersatz: Besteuerung eines Evangelisten, für dessen Unterhalt seine Glaubensfreunde sorgen.",
+				"court": "Tribunal Fédéral Suisse",
+				"docketNumber": "ATF 81 I 63",
+				"url": "https://www.bger.ch/ext/eurospider/live/fr/php/clir/http/index.php?lang=fr&type=highlight_simple_query&page=1&from_date=&to_date=&from_year=1954&to_year=2021&sort=relevance&insertion_date=&from_date_push=&top_subcollection_clir=bge&query_words=UNterhalt&part=all&de_fr=&de_it=&fr_de=&fr_it=&it_de=&it_fr=&orig=&translation=&rank=1&highlight_docid=atf%3A%2F%2F81-I-63%3Ade&number_of_ranks=714&azaclir=clir",
```

Comment:
OK, final nitpick. Can we clean these URLs up a bit? Item URLs should be as close to concise permalinks as possible, with parameters like search queries and filters removed. Here, for instance, we seem to be able to get away with https://www.bger.ch/ext/eurospider/live/fr/php/clir/http/index.php?lang=fr&type=highlight_simple_query&highlight_docid=atf%3A%2F%2F81-I-63%3Ade - much shorter and more appropriate for a printed bibliography.

Reply:
Although federal court decisions are never referenced by URL in a bibliography (in Switzerland, at least), I agree it's common sense and much better looking. Done.